plugins/m/test/code/console-colors.ansi

```diff
@@ -1,5 +1,99 @@
+  [48;5;160m160[0m [48;5;161m161[0m [48;5;162m162[0m [48;5;163m163[0m [48;5;164m164[0m [48;5;165m165[0m [48;5;166m166[0m [48;5;167m167[0m [48;5;168m168[0m [48;5;169m169[0m [48;5;170m170[0m [48;5;171m171[0m [48;5;172m172[0m [48;5;173m173[0m [48;5;174m174[0m [48;5;175m175[0m
+  [48;5;176m176[0m [48;5;177m177[0m [48;5;178m178[0m [48;5;179m179[0m [48;5;180m180[0m [48;5;181m181[0m [48;5;182m182[0m [48;5;183m183[0m [48;5;184m184[0m [48;5;185m185[0m [48;5;186m186[0m [48;5;187m187[0m [48;5;188m188[0m [48;5;189m189[0m [48;5;190m190[0m [48;5;191m191[0m
+  [48;5;192m192[0m [48;5;193m193[0m [48;5;194m194[0m [48;5;195m195[0m [48;5;196m196[0m [48;5;197m197[0m [48;5;198m198[0m [48;5;199m199[0m [48;5;200m200[0m [48;5;201m201[0m [48;5;202m202[0m [48;5;203m203[0m [48;5;204m204[0m [48;5;205m205[0m [48;5;206m206[0m [48;5;207m207[0m
+  [48;5;208m208[0m [48;5;209m209[0m [48;5;210m210[0m [48;5;211m211[0m [48;5;212m212[0m [48;5;213m213[0m [48;5;214m214[0m [48;5;215m215[0m [48;5;216m216[0m [48;5;217m217[0m [48;5;218m218[0m [48;5;219m219[0m [48;5;220m220[0m [48;5;221m221[0m [48;5;222m222[0m [48;5;223m223[0m
+  [48;5;224m224[0m [48;5;225m225[0m [48;5;226m226[0m [48;5;227m227[0m [48;5;228m228[0m [48;5;229m229[0m [48;5;230m230[0m [48;5;231m231[0m [48;5;232m232[0m [48;5;233m233[0m [48;5;234m234[0m [48;5;235m235[0m [48;5;236m236[0m [48;5;237m237[0m [48;5;238m238[0m [48;5;239m239[0m
+  [48;5;240m240[0m [48;5;241m241[0m [48;5;242m242[0m [48;5;243m243[0m [48;5;244m244[0m [48;5;245m245[0m [48;5;246m246[0m [48;5;247m247[0m [48;5;248m248[0m [48;5;249m249[0m [48;5;250m250[0m [48;5;251m251[0m [48;5;252m252[0m [48;5;253m253[0m [48;5;254m254[0m [48;5;255m255[0m
+
+[1mBackground colors do not heed or change brightness flag[0m
+  [1mBright [102mBrightGreen [42mGreen[0m
+  [102mBrightGreen [30mBlack[0m
+
+[1mForeground and background colors stack:[0m
+  [34;43mBlue on Yellow[0m
+  [38;5;12;103mPalette 12 on BrightYellow[0m
+  [37;48;5;88mWhite on Palette 88[0m
+  [97;48;2;128;0;128mBrightWhite on RGB 128,0,128[0m
+  [38;2;0;128;128;48;2;0;128;128mRGB 0,128,128 on RGB 0,128,128[0m
+  [38;2;192;192;0;48;2;0;128;128mRGB 192,192,0 on RGB 0,128,128[0m
+
+[1mOther commands are ignored[0m
+  [2mhalf-bright[0m
+  [4mset underscore[0m
+  [5mset blink[0m
+  [7mset reverse video[0m
+  [10mreset selected mapping, display control flag, and toggle[0m
+  [11mselect null mapping, set display control flag, reset[0m
+  [12mselect null mapping, set display control flag, set toggle[0m
+  [21mset underline[0m
+  [24munderline off[0m
+  [25mblink off[0m
+  [27mreverse video off[0m
```

Comment:
That's not easy to read. The relevant section of `page.html` looks like this

![image](https://user-images.githubusercontent.com/3275797/82854597-e3c8d080-9ed6-11ea-9be3-5d3fffa2119e.png)